packages/x-markdown/src/XMarkdown/index.css

```
@@ -207,3 +207,457 @@ xmd-tail {
+.x-markdown.x-markdown-light th,
+.x-markdown.x-markdown-dark th,
+.x-markdown.x-markdown-light td,
+.x-markdown.x-markdown-dark td {
+  padding: var(--td-th-padding);
+  border: 0;
+  text-align: left;
+  vertical-align: top;
+  max-width: var(--markdown-table-cell-max-width);
+  white-space: normal;
+  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.6.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)